linode_api4/objects/object_storage.py

```diff
@@ -478,4 +501,5 @@ class ObjectStorageKeys(Base):
         "secret_key": Property(),
         "bucket_access": Property(),
         "limited": Property(),
+        "regions": Property(),
```

Comment:
Could we mark this as unordered=True? That should fix a downstream diffing issue I'm seeing in Ansible 🙂

Reply:
Nice found! Fixed.